website/docs/guides/markdown-features/markdown-features-admonitions.mdx

```diff
@@ -121,3 +121,35 @@ import TabItem from '@theme/TabItem';
 ```
 
 :::
+
+## Custom Admonitions
+
+If you're using `@docusaurus/preset-classic`, you can create custom admonitions by adding them to your website's `docusaurus.config.js`. 
+
+For example, the following configuration adds an admonition with the `custom` keyword. This admonition will be styled as an [Infima `alert-info` component](https://infima.dev/docs/components/alert):
```

Comment:
I'd rather not couple this doc to Infima, considering we'll provide themes that are not using Infima in the future.

What you can say is that the admonition will have a `.admonition-custom` classname that users can target with custom CSS